.ci/infra/terraform/main.tf

```diff
@@ -259,6 +259,7 @@ module "project-services" {
     "iap.googleapis.com",
     "identitytoolkit.googleapis.com",
     "ids.googleapis.com",
+    "integrations.googleapis.com",
```

Comment:
apologies again, we need to remove this for now. I can add it in a separate PR, it's not necessary to add at this moment.

Reply:
Done